Vagrantfile

```diff
@@ -95,6 +95,12 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
       app.vm.synced_folder "src/rf/apps", "/opt/app/apps"
     else
       app.vm.synced_folder "src/rf", "/opt/app/"
+      app.vm.synced_folder "#{ENV['HOME']}/.aws", "/var/lib/rf/.aws/",
+        owner: "rf",
```

Comment:
I don't think this will work on fresh provisions because the `rf` user won't exist on the virtual machine. Was there a permissions issue you hit that caused you to add these?